templates/patterns/list_item/pattern-list-item--variant-date.html.twig

```diff
@@ -28,8 +28,27 @@
   {% endblock %}
   <div class="ecl-u-flex-grow-1">
     {% block meta %}
-      {% if meta is not empty %}
-        <div class="ecl-content-item-date__meta ecl-u-type-s ecl-u-type-color-grey-75 ecl-u-mb-xs">{{ meta|filter_empty|safe_join(" | ") }}</div>
+      {% if meta is not empty or status is not empty or highlighted %}
+        <div class="ecl-content-item-date__meta ecl-u-type-s ecl-u-type-color-grey-75 ecl-u-mb-xs">
+          {% if highlighted %}
+            {% include '@ecl-twig/label' with {
+              'label': 'Highlighted'|t,
+              'variant': 'highlight',
+              'extra_classes': 'ecl-u-mr-xs',
+            } only %}
+          {% endif %}
+          {% if status is not empty %}
+            <span id="status">
```

Comment:
I would not use an ID here, better if we add a class like "ecl-content-item-date__status" or smth